### DISCUSSION TITLE
Circular node connection

### DISCUSSION THREAD

**Discussion by SudeshAK** [hizzgdev/jsmind#612] [General]
Is it possible to create screens as attached in jsmind? 
![1718006211172139979248361503207](https://github.com/hizzgdev/jsmind/assets/36342286/38525d0d-3752-4fce-8404-ff53996b23c8)
![17180063340728566781477379177944](https://github.com/hizzgdev/jsmind/assets/36342286/593bf2dc-71b2-48eb-912f-0df551c52b64)

**Comment by hizzgdev** [comment]
Thanks for the support on the jsMind project!
Unfortunately, the current data structure may not cut it for you since jsMind is mainly about expanding nodes and doesn't handle converging ones.
Still, really appreciate the feedback, and we'll look into how to improve for a wider audience. If you have any further questions or suggestions, please don't hesitate to let us know. Looking forward to hooking you up better in the future! Thanks!
